examples/src/main/java/io/dapr/springboot/OpenTelemetryInterceptor.java

```diff
@@ -37,15 +39,19 @@ public boolean preHandle(
 
     Span span;
     try {
-      Context context = textFormat.extract(
-          Context.current(),
-          request,
-          new HttpTextFormat.Getter<HttpServletRequest>() {
-            @Override
-            public String get(HttpServletRequest req, String key) {
-              return req.getHeader(key);
-            }
-          });
+      TextMapPropagator.Getter<HttpServletRequest> getter = new TextMapPropagator.Getter<HttpServletRequest>() {

```

Comment:
Please, make this getter a static final field too.

Reply:
Done